src/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './io-ts';
```

Comment:
what about naming it `remote-data-io.ts`?

Reply:
Sure.